test/Microsoft.Sbom.Targets.Tests/AbstractGenerateSbomTaskInputTests.cs

```diff
@@ -226,9 +256,9 @@ public void Sbom_Generation_Succeeds_For_Null_Verbosity()
     public void Sbom_Generation_Succeeds_For_Invalid_Verbosity()
     {
         // Arrange
-        // If an invalid Verbosity is specified, the default value should be Verbose and is printed in the
-        // tool's standard output.
-        var pattern = new Regex("Verbosity=.*Value=Verbose");
+        // If an invalid Verbosity is specified, the default value should be Verbose. It is also printed in the
```

Comment:
nit: in the comment, replace "the default value should be Verbose" with "the default value should be Information"

Reply:
Addressed at e968302eefdfd075248b6a9aa6c4eb0e7694b4b3